editor/debugger/script_editor_debugger.cpp

```diff
@@ -1192,6 +1192,23 @@ void ScriptEditorDebugger::_method_changed(Object *p_base, const StringName &p_n
 	}
 
 	if (node) {
+		// Do not send the root transform of edited scene. Without this exception, all instances of the scene
+		// in the running game will snap to the same location, which is not desirable. Even in the case where
+		// there is only a single instance in the running game, its position within its parent scene still
+		// takes priority over its root position within its own scene. The subnodes in the parent scene can be
+		// edited instead to achieve the same goal.
```

Comment:
Does this comment to be so lengthy?
```suggestion	
		// Do not send the root transform of edited scene, because it affects all instances in undesirable way.
```

Reply:
That was because of this review:
https://github.com/godotengine/godot/pull/86659#discussion_r1440725295

Reply:
:/
You could at least avoid repeating it in 2 places. Or maybe you could leave a link to this PR for further explanation, we do that sometimes, e.g.
https://github.com/godotengine/godot/blob/89cc635c0554cb2e518c830969ca4c5eedda0f4e/editor/editor_themes.cpp#L345

Reply:
Are there best practices I can refer to somewhere about leaving comments in the codebase?

Reply:
There's this:
https://docs.godotengine.org/en/stable/contributing/development/code_style_guidelines.html#comment-style-guide

Reply:
Okay, I've amended.